vpr/SRC/route/route_timing.c

```diff
@@ -65,13 +83,18 @@ struct more_sinks_than {
 #ifdef PROFILE
 static void congestion_analysis();
 static void time_on_criticality_analysis();
-constexpr int fanout_per_bin = 4;
+constexpr unsigned int fanout_per_bin = 4;
 constexpr float criticality_per_bin = 0.05;
 static vector<float> time_on_fanout;
-static vector<float> time_to_build_heap;
-static vector<int> itry_on_fanout;
+static vector<float> time_on_fanout_rebuild;
 static vector<float> time_on_criticality;
+static vector<int> itry_on_fanout;
 static vector<int> itry_on_criticality;
+static vector<int> rerouted_sinks;
+static vector<int> finished_sinks;
+static int entire_net_rerouted;
+static int entire_tree_pruned;
+static int part_tree_preserved;
 #endif

```

Comment:
Should have a comment on what profiling does, and brief comments on the data variables here.